test/lib/clients/oidc_test.rb

```diff
@@ -0,0 +1,32 @@
+require "test_helper"
+
+# NOTE: these tests excercise the OIDC config but can't really verify a
+# successful OIDC login.  (Because that requires browser interaction.)
+# See the readme for how to use oidc login with the browser.
+
+class OIDCTest < ActiveSupport::TestCase
+  setup do
+    @client = Clients::Vault
+    @client.configure_oidc_user(@client.initial_user[:name],
+                                @client.initial_user[:email], test_policy)
+    @entity = @client.read_entity(@client.initial_user[:name])
+  end
+
+  test "#policies_contain_initial_users_email" do
```

Comment:
this implies a method name like this exists and you are testing it -- but really it's just a description, so no "#" and underscores needed